### PR TITLE
opencv: Add VTK option

### PR DESCRIPTION
**Pull request by mherb**
VTK is required for the opencv_viz module

### CODE REVIEW

File: opencv.rb [hunk 1]
```diff
@@ -22,6 +22,7 @@ class Opencv < Formula
   option "with-quicktime", "Use QuickTime for Video I/O instead of QTKit"
   option "with-opengl", "Build with OpenGL support"
   option "with-ximea", "Build with XIMEA support"
+  option "with-vtk", "Build with VTK support"
```
Comment (dpo) [inline opencv.rb]: This is not necessary, and is redundant with line 50.

**Comment by dpo** [comment]
Please add a revision number to the formula.

**Comment by mherb** [comment]
I am somewhat unsure why this keeps failing..

```
==> Fetching butterflow from homebrew/science
==> Downloading https://homebrew.bintray.com/bottles-science/butterflow-0.2.0.mavericks.bottle.tar.gz
Already downloaded: /Library/Caches/Homebrew/butterflow-0.2.0.mavericks.bottle.tar.gz
SHA1: a1e58f4a0b3b96cc82e6fa343ed52dd3479064e8
SHA256: 5a34d80342c4621ea4f98f15c04a54e5b3acab6a0112a8d7788789d615656d4f
==> Retrying download
==> Downloading https://homebrew.bintray.com/bottles-science/butterflow-0.2.0.mavericks.bottle.tar.gz
Warning: Formula reports different SHA256: 0b5be6c1f6ff5df7659b213ba250784bfd53d4f1ef5c4fa03f5221c21497a80b
Downloaded to: /Library/Caches/Homebrew/butterflow-0.2.0.mavericks.bottle.tar.gz
SHA1: a1e58f4a0b3b96cc82e6fa343ed52dd3479064e8
SHA256: 5a34d80342c4621ea4f98f15c04a54e5b3acab6a0112a8d7788789d615656d4f
```

File: opencv.rb [hunk 2]
```diff
@@ -4,6 +4,7 @@ class Opencv < Formula
   url "https://github.com/Itseez/opencv/archive/2.4.12.tar.gz"
   sha256 "8989f946a66fa3fc2764d637b1c866caf28d074ece187f86baba66544054eefc"
   head "https://github.com/Itseez/opencv.git", :branch => "2.4"
+  revision 2
```
Comment (dpo) [inline opencv.rb]: `revision 1` would do ;-)

Comment (mherb) [inline opencv.rb]: I wasn't sure about that because the bottles already seem to be revision 1..
Sorry about that ;)

**Comment by dpo** [comment]
Hopefully #3190 will fix the `butterflow` issue. I can reproduce it here.

**Comment by dpo** [comment]
@BrewTestBot test this please

**Pull request closed without merging**